test/com/puppetlabs/http/client/impl/metrics_unit_test.clj

```diff
@@ -0,0 +1,150 @@
+(ns com.puppetlabs.http.client.impl.metrics-unit-test
```

Comment:
I'm not sure that this is the right place for these tests, because there are some tests of the clojure code here in addition to the java classes. I'm also not sure how much that matters.